src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ClientRepository.cs

```diff
@@ -208,6 +224,19 @@ public async Task DeleteClientAsync(Guid clientId)
             }
         }
 
+        public async Task DeleteMemberClientsWithThresholdAsync(Guid memberId, int inactiveHoursThreshold)
+        {
+            using (var connection = _sqlConnectionFactory.CreateConnection())
+            {
+                var sqlQuery = $@"
+                    DELETE FROM Clients 
+                    WHERE 
+                        {nameof(Client.MemberId)} = @{nameof(memberId)} AND DATEDIFF(MINUTE, {nameof(Client.LastClientActivity)}, GETUTCDATE()) > @{nameof(inactiveHoursThreshold)} * 60";
```

Comment:
I think that if you decided to convert hours into minutes than better to replace inactiveHoursThreshold with inactiveMinutesThreshold.

Reply:
replaced